custom_metric.py

```
@@ -0,0 +1,329 @@
+from deepeval.metrics.utils import (
+    construct_verbose_logs,
+    trimAndLoadJson,
+    check_llm_test_case_params,
+    initialize_model,
+)
+from deepeval.test_case import (
+    LLMTestCase,
+    LLMTestCaseParams,
+    ConversationalTestCase,
+)
+from deepeval.metrics import BaseMetric
```

Ruff (0.8.2)

Redefinition of unused `BaseMetric` from line 2

Remove definition: `BaseMetric`

(F811)